src/component/movies/MovieCommentList.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import MovieCommendCard from './MovieCommentCard';
+
+const MovieCommentList = ({ comments }) => (
+  <div>
+    <div className="d-flex">
+        <div className="col">
+          { comments.map((comment, index) => <MovieCommendCard key={ index } comment={ comment } /> )}
```

Comment:
ako imas id na commnet-u onda stavi njega kao key, uglavnom bolje je radi re-rednerovanja i par stvari jos , mozes procitati detaljnije o tome ovdje https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
Nemam id comment-a, ali mogu prosiriti serializer na BE da dodatno vraca i id pa da onda upotrebim id. Citao sam bas ranije taj deo dokumentacije bas, ali posto nisam vracao id zato sam index upotrebio.

Reply:
vrati i id